docs/detections/prebuilt-rules/prebuilt-rules-reference.asciidoc

```diff
@@ -752,6 +774,10 @@ and their rule type is `machine_learning`.
 
 |<<potential-lsa-authentication-package-abuse, Potential LSA Authentication Package Abuse>> |Adversaries can use the autostart mechanism provided by the Local Security Authority (LSA) authentication packages for privilege escalation or persistence by placing a reference to a binary in the Windows registry. The binary will then be executed by SYSTEM when the authentication packages are loaded. |[Elastic] [Host] [Windows] [Threat Detection] [Privilege Escalation]  |7.12.0 |1
 
+|<<potential-lsass-clone-creation-via-psscapturesnapshot, Potential LSASS Clone Creation via PssCaptureSnapShot>> |Identifies the creation of an LSASS process clone via PssCaptureSnapShot where the parent process is the initial LSASS process instance. This may indicate an attempt to evade detection and dump LSASS memory for credential access. |[Elastic] [Host] [Windows] [Threat Detection] [Credential Access]  |8.0 |1
+
+|<<potential-lsass-memory-dump-via-psscapturesnapshot, Potential LSASS Memory Dump via PssCaptureSnapShot>> |Identifies suspicious access to an LSASS handle via PssCaptureSnapShot where two successive process access are performed by the same process and targeting two different instances of LSASS. This may indicate an attempt to evade detection and dump LSASS memory for credential access. |[Elastic] [Host] [Windows] [Threat Detection] [Credential Access]  |8.0 |1
```

Comment:
```suggestion
|<<potential-lsass-clone-creation-via-psscapturesnapshot, Potential LSASS Clone Creation via PssCaptureSnapShot>> |Identifies the creation of an LSASS process clone via PssCaptureSnapShot where the parent process is the initial LSASS process instance. This may indicate an attempt to evade detection and dump LSASS memory for credential access. |[Elastic] [Host] [Windows] [Threat Detection] [Credential Access]  |8.0.0 |1

|<<potential-lsass-memory-dump-via-psscapturesnapshot, Potential LSASS Memory Dump via PssCaptureSnapShot>> |Identifies suspicious access to an LSASS handle via PssCaptureSnapShot where two successive process access are performed by the same process and targeting two different instances of LSASS. This may indicate an attempt to evade detection and dump LSASS memory for credential access. |[Elastic] [Host] [Windows] [Threat Detection] [Credential Access]  |8.0.0 |1
```